GameDefinition/Content/ItemKindActor.hs

```diff
@@ -474,6 +474,24 @@ alligator = ItemKind
                , ("eye 6", COrgan), ("ear 9", COrgan)
                , ("animal brain", COrgan) ]
   }
+octopus = ItemKind
+  { isymbol  = 'o'
+  , iname    = "Giant octopus"
```

Comment:
Would you like it to be a unique monster (at most one occurrence in the game)? The convention is, for uniques, we capitalize all, for non-uniques, we use the capitalization from Wikipedia.

Reply:
I thought `Giant octopus` is the capitalization from Wikipedia. But should I change it to `giant octopus`? I don't want it to be unique.

Reply:
Indeed, they are not consistent in capitalization. Let's pick `giant octopus`; I think it's the more common one.